corehq/apps/app_manager/static/app_manager/js/forms/form_designer.js

```diff
@@ -209,5 +210,16 @@ hqDefine("app_manager/js/forms/form_designer", function() {
         $("#edit-form-name-modal button[type='submit']").click(function() {
             hqImport('analytix/js/kissmetrix').track.event("Renamed form from form builder");
         });
+
+        // if they are in the guided tour, the preview should pop out after 3 minutes
+        // TODO: improve usage.
```

Comment:
Agreed, we should figure out something different here. I was imagining logic like, "On save, pop out app preview, unless during this page load (or session?) the user has already manually close app preview." ...but let's talk with Marshall about this.

Reply:
This should probably also check that Appcues is in use, so it doesn't happen for user who aren't in their first 30 days.

Reply:
Yeah, Marshall mentioned thinking about some time based thing that would say "nice app, try it out in app preview". Definitely happy to catch up on this

Reply:
`initial_page_data("guided_tour")` seemed like a reasonable proxy, but that might have been a bad assumption. When is the guided tour shown?

Reply:
Ha, very reasonable guess, but `guided_tour` is dead code. It's leftover from when we released case property saving in the form builder, there was a guided tour for a while to introduce people to the new functionality. I didn't realize it was still here, should do a PR to clean it up.